### ISSUE TITLE
New Component: SIP / WebRTC client

### ISSUE THREAD

**Issue by jepsar** [primefaces-extensions/primefaces-extensions#127]
For example using: https://sipjs.com/

**Comment by jepsar** [comment]
FYI I looks like that we've received an okay to add a SIP interface to one of our projects, so I think I can start work on this in the coming weeks.

**Comment by melloware** [comment]
OK I assigned it to you.

**Comment by jepsar** [comment]
Seems there is a delay / shift in priority at our client.. but they want this at some point

**Comment by rider87** [comment]
@jepsar do you have an update here :) ?

**Comment by jxmai** [comment]
Wondering if this is still the ideal future chat component after these years since the website demo seems broken?

Go to: https://sipjs.com/

Type "Hello Bob!" on Alice view, and click send, nothing seems happened on web UI, and on console, I can see `SIP/2.0 404 Not Found`

Before sent:

![image](https://github.com/user-attachments/assets/f0750ff9-be4d-4e53-bfb8-1c4cd64e425f)

---

After sent:

![image](https://github.com/user-attachments/assets/994d2cd2-294d-45ca-8b58-d2ae8e955537)

**Comment by melloware** [comment]
Probably not unless its just their infrastructure is down.  Although last commit and release was 2 years ago.

**Comment by jxmai** [comment]
Yeah, it appears the last release from SIP.js https://github.com/onsip/SIP.js/releases/tag/0.21.2 was two years ago, with no updates since then. I am curious about the long-term goal for this component, epecially since the related request in PF Core https://github.com/primefaces/primefaces/issues/8516 was redirected to this ticket.

Do we want to build a message component like vaadin? https://vaadin.com/docs/latest/components/message-input

**Comment by melloware** [comment]
this one is more modern and updated 6 months ago: https://jssip.net/

I think we just need to find the right library, with the RIGHT open source license ,that is still active.

**Comment by jxmai** [comment]
Let's see what's the thought from @jepsar as well since he's the intial author of the ticket.

**Comment by jepsar** [comment]
I'm not bound to a certain implementation. We recently started using Vonage. So far I haven't had the need to create a component for it.
